### PR TITLE
fix function/end keywords not applied for every scope

### PR DESCRIPTION
**Pull request by kyazdani42**

### CODE REVIEW

File: queries/lua/highlights.scm [hunk 1]
```diff
@@ -100,9 +100,13 @@
...
-("function" @keyword.function
- [(function_name) (identifier)] @function
- "end" @keyword.function)
+(function [(function_name) (identifier)] @function)
+(function ["function" "end"] @function)
+
+(local_function [(function_name) (identifier)] @function)
+(local_function ["function" "end"] @function)
+
+(function_definition ["function" "end"] @function)
```
Comment (stsewd) [inline queries/lua/highlights.scm:109]: We are capturing these with `@keyword.function`

Comment (vigoux) [inline queries/lua/highlights.scm:109]: Use `kwyword.function` here please.

**Review by vigoux — CHANGES_REQUESTED** [review]

File: queries/lua/highlights.scm (hunk 1 above)
Comment (kyazdani42) [reply queries/lua/highlights.scm:109]: woups

Comment (kyazdani42) [reply queries/lua/highlights.scm:109]: woupsy

**Review by stsewd — APPROVED** [review]

**Review by vigoux — APPROVED** [review]

**Pull request merged**